docs/attributes-registry/clr.md

```diff
@@ -0,0 +1,15 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# CLR
+
+## Clr Attributes
```

Comment:
```suggestion
## CLR Attributes
```

Reply:
This one is autogenerated, and it doesn't use the abbreviation here. The same occurs in the JVM attributes. @trisch-me, do you have any thoughts on whether this is a generator bug?

Reply:
yeah, I would call it a bug. 
Fix it here #1043

Reply:
It is merged. You can regenerate snippets